Source/WebGPU/WebGPU/Texture.h

```diff
@@ -84,7 +84,15 @@ class Texture : public WGPUTextureImpl, public RefCounted<Texture> {
     WGPUExtent3D physicalMiplevelSpecificTextureExtent(uint32_t mipLevel);
 
     id<MTLTexture> texture() const { return m_texture; }
-    const WGPUTextureDescriptor& descriptor() const { return m_descriptor; }
+
+    uint32_t width() const { return m_width; }
+    uint32_t height() const { return m_height; }
+    uint32_t depthOrArrayLayers() const { return m_depthOrArrayLayers; }
+    uint32_t mipLevelCount() const { return m_mipLevelCount; }
+    uint32_t sampleCount() const { return m_sampleCount; }
+    WGPUTextureDimension dimension() const { return m_dimension; }
+    WGPUTextureFormat format() const { return m_format; }
+    WGPUTextureUsageFlags usage() const { return m_usage; }
```

Comment:
What does this really win us?

Reply:
Just matching the spec. https://gpuweb.github.io/gpuweb/#texture-interface

Reply:
If you think that's worth it, I'll trust your judgement